tensorflow_datasets/structured/somerville_happiness.py

```diff
@@ -0,0 +1,95 @@
+"""Somerville Happiness Survey Data Set from UCI Machine Learning Repository"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import csv
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """
+@misc{Waldemar:2020 ,
+author = "Waldemar W. Koczkodaj",
+year = "2015",
+title = "{UCI} Machine Learning Repository",
+url = "http://archive.ics.uci.edu/ml",
```

Comment:
this is the URL to the "global" page - could you point to the url of this dataset ?